bin/main.rb

```diff
@@ -1,2 +1,100 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+
+# rubocop :disable Metrics/BlockNesting
+
+def display_board(board_cells)
+  puts "#{board_cells[1]} | #{board_cells[2]} | #{board_cells[3]}"
+  puts '__|___|__'
+  puts '  |   |'
+  puts "#{board_cells[4]} | #{board_cells[5]} | #{board_cells[6]}"
+  puts '__|___|__'
+  puts '  |   |'
+  puts "#{board_cells[7]} | #{board_cells[8]} | #{board_cells[9]}"
+  puts ''
+end
+
+puts 'Wellcome to the tic-tac-toe geme!'
+
+play_again = true
+while play_again
+
+  board = ['', 1, 2, 3, 4, 5, 6, 7, 8, 9]
+  display_board(board)
+  input_symb1 = ''
+  until %w[X O].include?(input_symb1)
+    puts 'Please choose between X and O to play with.'
+    input_symb1 = gets.chomp.upcase! # Read player option from the user.
+  end
+  puts "You have chosen #{input_symb1} as your game symbol."
+  input_symb2 = if input_symb1 == 'O'
+                  'X'
+                else
+                  'O'
+                end
+
+  puts "Your opponent will play with the game #{input_symb2}"
+
+  puts ''
+  winner = false
+  played = false
+
+  count = 0
+
+  while count < 9 && !winner
```

Comment:
:warning: Comparing with a sudden number indicates that there is a problem based on scalability and feasibility. These are also the terms of single responsibility principles. I think you don't have the intension to use this condition on your game logic milestone.